packages/karma-esm/src/forked-babel-plugin-bare-import-rewrite.js

```diff
@@ -0,0 +1,163 @@
+// @ts-nocheck
+/* eslint-disable */
+'use strict';
+
+/**
+ * Fork of babel-plugin-bare-import-rewrite:
```

Comment:
maybe use `patch-package` for this?

Reply:
I'd rather spend the effort making a PR to the babel plugin. I already did a few PRs, but I'd like to get this change merged so that people can start using it.